Makefile

```diff
@@ -74,9 +74,13 @@ test: manifests generate fmt vet ## Run tests.
 ##@ Deployment
 
 .PHONY: deploy
-deploy: manifests ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+deploy: manifests cert-manager ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	$(KAPP) deploy -a $(KAPP_APP) -n $(KAPP_APP_NAMESPACE) -c -f config/kapp -f <($(KO) resolve --platform $(KO_PLATFORMS) -f config/servicebinding-runtime.yaml)
 
+.PHONY: cert-manager
+cert-manager:
+	$(KAPP) deploy -a $(KAPP_APP) --wait-timeout 5m -y -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
```

Comment:
Looks like the cert-manager is deployed in the `cert-manager` namespace. But that's not explicitly mentioned here. Let me know if I should specify the namespace as `cert-manager`.

Reply:
Namespaces for apps is a bit chicken<->egg. You cannot create the app in a namespace that doesn't exist. If you create the namespace to create the app within it, and the manifest contains that same namespace, it will fail because you're asking kapp to own a namespaces that already exists.

I usually end up being lazy and create everything in the default namespace for dev cluster. For important clusters, I'll create a namespace dedicated to holding these app references. The resource is a ConfigMap, so name collisions are possible.